airbyte/caches/snowflake.py

```diff
@@ -3,14 +3,52 @@
 
 ## Usage Example
 
+# Password connection:
+
+```python
+from airbyte as ab
+from airbyte.caches import SnowflakeCache
+
+cache = SnowflakeCache(
+    account="myaccount",
+    username="myusername",
+    password=ab.get_secret("SNOWFLAKE_PASSWORD"), # optional
+    warehouse="mywarehouse",
+    database="mydatabase",
+    role="myrole",
+    schema_name="myschema",
+)
+```
+
+# Private key connection:
```

Comment:
Thanks for adding these usage examples. They render automatically into our API docs using pdoc - so it's a huge help for future users.